bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/oven.xml

```diff
@@ -22,10 +22,21 @@
 			<channel id="duration" typeId="duration"/>
 			<channel id="elapsed" typeId="elapsed"/>
 			<channel id="finish" typeId="finish"/>
-			<channel id="target" typeId="target"/>
-			<channel id="measured" typeId="measured"/>
-			<channel id="temp1" typeId="temp"/>
-			<channel id="temp2" typeId="temp"/>
+			<channel id="target" typeId="targetTemperature">
+				<description>Target temperature to be reached by the oven</description>
+			</channel>
+			<channel id="measured" typeId="currentTemperature">
+				<label>Measured Temperature</label>
+				<description>Actual measured temperature in the oven</description>
+			</channel>
+			<channel id="temp1" typeId="temperature">
+				<label>Temperature</label>
+				<description>Program temperature in the oven</description>
+			</channel>
+			<channel id="temp2" typeId="temperature">
+				<label>Temperature</label>
+				<description>Program temperature in the oven</description>
+			</channel>
```

Comment:
Maybe use "Program Temperature" as label ?
And now way to give a hint of the difference between these 2 temperatures ?

Reply:
Existing labels, not changed. Unfortunately I don't know what they are as my oven doesn't support them. I have changed to "Program Temperature 1" and "Program Temperature 2".